### ISSUE TITLE
page__conditional-comment: Make template for conditional comments

### ISSUE THREAD

**Issue by tadatuta** [bem/bem-core#511]
As we decided to remove CC for `ieN.css` and we have https://github.com/bem/es5-shims now, I think it'd be useful to have element for conditional comments.

API:

``` js
{
    elem : 'cc',
    condition : 'lte IE 9',
    content : { elem : 'js', url : '//yastatic.net/es5-shims/0.0.1/es5-shims.min.js' }
}
```

template:

``` js
block('page').elem('cc')(
    tag()(false),

    content()(function() {
        var cond = this.ctx.condition,
            isNegation = cond.indexOf('!') > -1;

        return [
            '<!--[if ' + cond + ']>',
            isNegation ? '-->' : '',
            applyNext(),
            isNegation ? '<!--' : '',
            '<![endif]-->'
        ];
    })
);
```

// cc @mishanga @narqo

**Comment by tadatuta** [comment]
any consideration? should i create pr with this elem?

**Comment by narqo** [comment]
As es5-shims should be added for some Android devices as well, I don't have any strong opinion about this.

**Comment by sipayRT** [comment]
we need this elem for more beautiful adding styles for IE
@hcodes @narqo

**Comment by qfox** [comment]
+1

isNegation → hasNegation

**Comment by narqo** [comment]
Just in case...

``` js
{ block : 'page',
  head : [
    { elem : 'asset', 
      type : 'js', 
      url : '//yastatic.net/es5-shims/0.0.1/es5-shims.min.js' },

    { elem : 'asset', 
      type : 'text/json',    // ➡︎ <script type="text/json">···</script>
      url : '//yastatic.net/blah-blah-blah.json' },

    { elem : 'asset', 
      type : 'css', 
      url : '//yastatic.net/jquery-ui/1.11.2/themes/base/jquery-ui.min.css' },

    { elem : 'asset', 
      type : 'css', 
      ie : '<= 9'    // ➡︎ autoprefixer-way
      url : '//yastatic.net/jquery-ui/1.11.2/themes/base/jquery-ui.min.css' }
  ] }
```

**Comment by tadatuta** [comment]
@narqo looks reasonable but how'd you express CC for all browsers and special version of IE? (`'<!--[if gt IE 8]><!-->code from here will work both in normal browsers and IE > 8<!--<![endif]-->',`)

**Comment by narqo** [comment]
> [..] CC for all browsers and special version of IE?

I don't want even to think about such things :) Is there a big difference between `condition : '! lte IE 9'` in `page__cc` and `ie : '! lte IE 9'` in the variant above?

I think there could be a lot of awkward things, like `<scirpt type="application/x-javascript-template"`, which can be solved once, with simple content inlining. 

As I said: I still don't have any strong (well, positive) opinion about the topic :))

**Comment by veged** [comment]
Agree with such element in separate bemhtml-file and with `conditional-comment` name.

**Comment by sipayRT** [comment]
ping?

**Comment by narqo** [comment]
Fixed with #869

**Comment by narqo** [comment]
v3 port #912
